weave

```diff
@@ -115,10 +115,35 @@ docker_run_options() {
     echo --privileged --net=host $(docker_sock_options)
 }
 
+docker_exec_cni_options() {
```

Comment:
Also, that way any other commands which might need extra options can simply be handled with a `case` in `exec_remote_docker_run_options`.